docs/design_proposals/sync-absolute-paths.md

```diff
@@ -0,0 +1,60 @@
+# Sync support absolute paths
+
+* Author(s): Mustapha Yamie (@mustaphagit)
+* Design Shepherd: 
+* Date: 22/06/2021
+* Status: [In review]
+## Background
+
+Currently skaffold does not support **absolute paths** in `sync` section. For example, if we want to sync the file in `/home/config/**` to container's `/config` folder we can't do that. Currently skaffold only supports relative paths.
+
+Here is an example snippet for a new feature:
+___
+```yaml
+sync:
+  manual:
+    - src: '/home/config/**'
+      dest: /config/
+      type: absolute
```

Comment:
Is the `type` field necessary for supporting absolute paths?  Is it possible to use something like [filepath.IsAbs(<path>)](https://golang.org/pkg/path/filepath/#IsAbs) and infer if a path is absolute?

Reply:
Are there any edge cases where the path is absolute but the path does not being with "/"?   This is the IsAbs code:
```
// IsAbs reports whether the path is absolute.
func IsAbs(path string) bool {
	return len(path) > 0 && path[0] == '/'
}
```

Reply:
If path doesn't start with `/` the path is not absolute.

Reply:
And you are right , we don't need Type field actually. I deleted it and checked path type with `filepath.IsAbs()`